specifications/config.json

```diff
@@ -0,0 +1,127 @@
+{
+  "client": {
+    "apiKey":{
+      "file":"",
+      "id":"",
+      "secret":""
+    },
+    "cacheManager":{
+      "defaultTtl": 300,
+      "defaultTti": 300,
+      "caches": {
+        "account": {
+          "ttl": 300,
+          "tti": 300
+        }
+      }
+    },
+    "baseUrl": "https://api.stormpath.com/v1",
+    "connectionTimeout": 30,
+    "authenticationScheme": "SAUTHC1", // or "BASIC"
+    "proxy":{
+      "port": "",
+      "host": "",
+      "username": "",
+      "password": ""
+    }
+  },
+  "application": {
+    "name": "test",
+    "href": "https://..."
+  },
+  "web": {
+    "oauth2":{
+      "enabled": false,
+      "uri": "/oauth/token",
+      "password":{
```

Comment:
This and `client_credentials` will probably go away when the Stormpath API endpoints for this exist, correct?  Since they'll be configuring that there and we can just pull it from the API?